mxnet/inference/docker/1.8/py3/sdk1.17.0/Dockerfile.neuron

```diff
@@ -10,8 +10,7 @@ LABEL com.amazonaws.sagemaker.capabilities.accept-bind-to-port=true
 # https://docs.aws.amazon.com/sagemaker/latest/dg/build-multi-model-build-container.html
 LABEL com.amazonaws.sagemaker.capabilities.multi-models=true
 
-ARG MMS_VERSION=1.1.4
-ARG MX_URL=https://aws-mx-pypi.s3-us-west-2.amazonaws.com/1.8.0/aws_mx-1.8.0-py2.py3-none-manylinux2014_x86_64.whl
+ARG MMS_VERSION=1.1.8
```

Comment:
nit: Why is line 106 on this dockerfile needed? I don't think it has any effect on the resultant docker image, because it doesn't upgrade the MMS version from the version defined in this ARG, but it is a redundant step.